### PR TITLE
replace %s by {}

### PR DESCRIPTION
**Pull request by counter-reverse**
%s is a c style, it used to be used in python 2 and it should not be used anymore in python 3.

### CODE REVIEW

**Comment by pnu-s** [comment]
@counter-reverse I think your changes make sense but it has gone wild because of conflicts. Could you please try to solve them ? (or simply remake the changes on a clean new branch ?) Thanks!

**Comment by counter-reverse** [comment]
This MR did not work because there were another bug that appears when google play has changed icon: https://github.com/Exodus-Privacy/exodus-core/issues/13 https://github.com/Exodus-Privacy/exodus-core/issues/8. Now the bug is fixed. We should make another compilation test on this MR.

**Comment by pnu-s** [comment]
@counter-reverse You'll first have to merge your branch with v1 to make sure you are getting the latest changes and fix the conflicts.

**Comment by counter-reverse** [comment]
Really merge? Not rebase?

File: exodus_core/analysis/static_analysis.py [hunk 1]
```diff
@@ -509,22 +509,22 @@ def print_apk_infos(self):
...
         print("=== Information")
-        print('- APK path: %s' % self.apk_path)
-        print('- APK sum: %s' % self.get_sha256())
-        print('- App version: %s' % self.get_version())
-        print('- App version code: %s' % self.get_version_code())
-        print('- App UID: %s' % self.get_application_universal_id())
-        print('- App name: %s' % self.get_app_name())
-        print('- App package: %s' % self.get_package())
-        print('- App permissions: %s' % len(permissions))
+        print('- APK path: {0}'.format(self.apk_path))
```
Comment (pnu-s) [inline exodus_core/analysis/static_analysis.py]: I think we should replace all `{0}` by `{}`, as the 0 is not useful here.

File: setup.py [hunk 2]
```diff
@@ -59,4 +59,4 @@ def is_exe(fpath):
     install_requires=install_requires,
     include_package_data=True,
     zip_safe=False,
-)
+)
```
Comment (pnu-s) [inline setup.py]: I think this is a mistake here, the line at the end of file shouldn't be removed.

File: exodus_core/analysis/static_analysis.py (hunk 1 above)
Comment (counter-reverse) [reply exodus_core/analysis/static_analysis.py]: Good idea. I use to believe I have to add a number in the brackets necessary due to an old example.

File: setup.py (hunk 2 above)
Comment (counter-reverse) [reply setup.py]: I will check that.

**Review by pnu-s — CHANGES_REQUESTED** [review]
See inline comments

**Comment by pnu-s** [comment]
@counter-reverse Could you just use `{}` directly? (instead of {1}, {0}, {fingerprint}, etc.)

**Comment by counter-reverse** [comment]
@simpnu I read this post: https://stackoverflow.com/questions/28204936/python-format-best-practices and I believed I had to use some conventions to format that depends of the context of the code.

Now I have already replaced any {1}, {0}, {fingerprint} by {}.

File: exodus_core/analysis/static_analysis.py [hunk 3]
```diff
@@ -497,31 +497,31 @@ def print_apk_infos(self):
...
+        print('- App permissions: {}'.format(len(permissions)))
         for p in permissions:
-            print('    - %s' % p)
+            print('    - {}'.format(p))
         print('- App libraries:')
         for l in libraries:
-            print('    - %s' % l)
-        print('- Certificates: %s' % len(certificates))
+            print('    - {}'.format(l))
+        print('- Certificates: {}' % len(certificates))
```
Comment (pnu-s) [inline exodus_core/analysis/static_analysis.py]: This should be `print('- Certificates: {}'.format(len(certificates)))`

**Comment by pnu-s** [comment]
@counter-reverse Please check my last comment

File: exodus_core/analysis/static_analysis.py (hunk 3 above)
Comment (counter-reverse) [reply exodus_core/analysis/static_analysis.py]: @simpnu it is right now.

**Pull request merged**